src/components/LoanCards/Buttons/LendAmountButton.vue

```diff
@@ -2,7 +2,7 @@
 	<lend-button
 		price="amountLeft"
```

Comment:
@egoroza this needs to be `:price="amountLeft"` to ensure the dynamic value of `amountLeft` is passed in.

Reply:
fixed with: https://github.com/kiva/ui/pull/3764